operators/multiclusterobservability/pkg/config/config.go

```diff
@@ -277,6 +285,21 @@ var (
 	MemoryLimitMB   = int32(1024)
 	ConnectionLimit = int32(1024)
 	MaxItemSize     = "1m"
+
+	BackupResourceMap = map[string]string{
```

Comment:
we should only need to handle the resources which created by users. So the 4 secrets created by our operator not required

Reply:
Removed them in the latest commit